index.html

```diff
@@ -10,12 +10,9 @@
   </head>
   <body>
     <div class="book">
-      <div>
-        <img
-          class="book-cover"
-          src="./images/book-cover.png"
-          alt="book-cover"
-        />
+      <div class="book-cover">
+        <img class="cover" src="./images/book-cover.png" alt="book-cover" />
```

Comment:
Alt should be an explanation of the image, in this case, could be "Cook cover". No need for any middle-scores 😄

Reply:
Fixed in PR #6